primers/web-archive-formats/index.md

```diff
@@ -187,4 +187,4 @@ From here, we can use ```wget``` to look at what gets played back:
   X-Archive-Wayback-Perf: {"IndexLoad":359,"IndexQueryTotal":359,"RobotsFetchTotal":1,"RobotsRedis":1,"RobotsTotal":1,"Total":371,"WArcResource":10}
   X-Archive-Playback: 1
   X-Page-Cache: MISS
-~~~
\ No newline at end of file
+~~~
```

Comment:
Hopefully this is not impactful — I edited in the GitHub UI and it looks like it changed something non-obvious here (probably a line ending?). If it’s a problem, I can edit locally and remove this change.